### PR TITLE
Typography theming

### PR DESCRIPTION
**Pull request by VLegakis**
Some color updates in the UIKit

### CODE REVIEW

File: src/theme/default/colors.ts [hunk 1]
```diff
@@ -42,10 +42,10 @@ const rawColors: RawColorsT = {
 
 const appColors: AppColorsT = {
   borderColor: rawColors.gray.bombay,
-  controlFocused: rawColors.gray.limedSpruce,
+  controlFocused: rawColors.gray.darkMud,
   error: rawColors.red.redOrange,
   success: rawColors.green.greenHaze,
-  text: rawColors.gray.limedSpruce,
+  text: rawColors.gray.darkMud,
```
Comment (Azein) [inline src/theme/default/colors.ts]: Was the previous value wrong?

Comment (VLegakis) [reply src/theme/default/colors.ts]: All the latest mocks have this color for text (as well as `HeadPanel` etc...)

Comment (VLegakis) [reply src/theme/default/colors.ts]: Resolving after discussion with @oyabun concluding in using 335414A

File: src/components/typography/typography.stories.tsx [hunk 2]
```diff
@@ -28,7 +28,7 @@ const subData = {
 
 const headers = (
   <div>
-    <H0>H1: Title Enourmous. The five boxing wizards jump quickly.</H0>
+    <H0>H0: Title Enourmous. The five boxing wizards jump quickly.</H0>
```
Comment (Azein) [inline src/components/typography/typography.stories.tsx]: There is no h0 DOM element. Name should be changed I think.

Comment (VLegakis) [reply src/components/typography/typography.stories.tsx]: So is the case for H1, H2 etc they don't exist. 
In any case propose a name and I can rename it.
Also, this is not related to this PR.

**Pull request merged**